docs/streams/getting-started-streams.md

```diff
@@ -0,0 +1,459 @@
+        subscription_name="test-subscription-1")
+    for i in range(10):
+        print("In ",i)
+        m1 = json.loads(subscriber.recv())  # Listen on stream for any receiving messages
+        msg1 = base64.b64decode(m1["payload"])
+        print(F"Received message '{msg1}' id='{m1['messageId']}'") # Print the received message
+        subscriber.send(json.dumps({'messageId': m1['messageId']})) # Acknowledge the received message
+
+
+createStream()
+
+# User enters choice.
+user_input = input("Type 'w' or '1' to write data. Type 'r' or '0' to read data: ")
+if user_input == "w" or user_input == '1':
+    sendData()
+elif user_input == "r" or user_input == '0':
+    receiveData()
+else:
+    print ("Invalid user input. Stopping program")
+```
+
+</TabItem>
+<TabItem value="js" label="JavaScript">
+
+```js
+// Connect to GDN.
+const jsc8 = require("jsc8");
+// Choose one of the following methods to access the GDN. API key is recommended.
+const client = new jsc8({url: "https://gdn.paas.macrometa.io", apiKey: "diana.payton_macrometa.com.mykey.eFEvSibM2pUqsaSoL0S54ubZ0kzAU3U6iWAXRiQDqB7PSClEYMhq3WI2gEbvxYxK9c6eb4", fabricName: '_system'});
+console.log("Authentication done!!...");
+
+// Get GeoFabric details.
+async function getFabric() {
+    try {
+      await console.log("Getting the fabric details...");
+      let result = await client.get();
+  
+      await console.log("result is: ", result);
+    } catch(e){
+      await console.log("Fabric details could not be fetched because "+ e);
+    }
+  }
+  
+  getFabric();
+
+  // Create global and local streams.
+  async function streams() {
+    try{
+      await console.log("Creating local stream...");
+      const stream_local = await client.createStream("testStream-local", true);
+
+      await console.log("Creating global stream...");
+      const stream_global = await client.createStream("testStream-global", false);
+
+    } catch(e){
+      await console.log("Streams could not be fetched because "+ e);
+    }
+}
+
+streams();
+
+// Publish messages to stream.
+async function streams() {
+    try {
+      await console.log("Creating local stream...");
+      await client.createStream("my-stream", true);
+
+      const stream = await client.getStream("my-stream", true);
+      await stream.publishMessage("Hello World");
+
+
+// DFP - This part was from the "advanced" portion.
+// How can we make it output this message when the code above is successful?
+    //  producer.on("message", (msg) => {
+    //    console.log(msg, "Sent Successfully");
+    //  });
+
+    } catch(e) {
+      await console.log("Publishing could not be done because "+ e);
+    }
+}
+
+streams()
+
+// Subscribe to stream
+async function getDCList() {
+    let dcListAll = await client.listUserFabrics();
+    //DFP - Error here - geo_fabric is not defined. I could not test Step 5. 
+    let dcListObject = await dcListAll.find(function(o) { return o.name === geo_fabric; });
```

Comment:
just add `const geo_fabric  = "_system"` just before line no 432 inside function. you can use any fabric name.

Reply:
Thanks so much, that worked!